src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs

```diff
@@ -82,7 +82,7 @@ public interface IPropertyBinderImplementation : IEnableLogger
     /// This can be a child property, for example <c>x =&gt; x.Foo.Bar.Baz</c> in which case
     /// the binding will attempt to subscribe recursively to updates in order to
     /// always get and set the correct property.</param>
-    /// <param name="signalViewUpdate">An observable, that when signaled, indicates that the view property
+    /// <param name="signalViewUpdate">An observable, that when signalled, indicates that the view property
```

Comment:
Signaled is US english which is generally where we have the code base btw

Reply:
I used an US English dictionary. Strange that it picked that up as incorrect.

Reply:
Yeah it seems to be correcting to UK english elsewhere with behaviour vs behavior etc

Reply:
Updated